extensions/eclipse-che-theia-plugin-ext/src/browser/che-factory-main.ts

```diff
@@ -10,7 +10,8 @@
 
 import { interfaces } from 'inversify';
 import { CheFactoryMain } from '../common/che-protocol';
-import { CheApiService, FactoryDto } from '../common/che-protocol';
+import { CheApiService } from '../common/che-protocol';
```

Comment:
```
import { CheFactoryMain , CheApiService } from '../common/che-protocol'; 
``` 
?

Reply:
Would be nice to have a plugin / extension for che-Theia to auto group imports like that.
Thanks!